Makefile

```diff
@@ -189,8 +189,7 @@ DATAPLANE_RUNNER_IMG                             ?=quay.io/openstack-k8s-operato
 DATAPLANE_NETWORK_CONFIG_TEMPLATE                ?=templates/single_nic_vlans/single_nic_vlans.j2
 DATAPLANE_SSHD_ALLOWED_RANGES                    ?=['192.168.122.0/24']
 DATAPLANE_CHRONY_NTP_SERVER                      ?=clock.redhat.com
-DATAPLANE_OVN_METADATA_AGENT_NOVA_METADATA_HOST  ?=127.0.0.1
-DATAPLANE_OVN_METADATA_AGENT_PROXY_SHARED_SECRET ?=12345678
+DATAPLANE_OVN_METADATA_AGENT_PROXY_SHARED_SECRET ?=42
```

Comment:
yeah that is fairly arbitrary. We eventually need to expose configuration for this via the CRDs. But this is the one that is hardcoded today on the nova-operator side.